pkg/kubectl/rolling_updater_test.go

```diff
@@ -672,7 +672,7 @@ func TestUpdateWithRetries(t *testing.T) {
 		Codec: codec,
 		Client: client.HTTPClientFunc(func(req *http.Request) (*http.Response, error) {
 			switch p, m := req.URL.Path, req.Method; {
-			case p == "/api/v1beta3/namespaces/default/replicationcontrollers/rc" && m == "PUT":
+			case p == "/api/"+latest.Version+"/namespaces/default/replicationcontrollers/rc" && m == "PUT":
```

Comment:
1. Shouldn't these be testapi.Version()?
2. Shouldn't these use testapi.ResourcePathWithPrefix and similar methods to generate the resource paths?